### PR TITLE
xzcat: add page 

### PR DESCRIPTION
**Pull request by TerribleCodes**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented: xz (XZ Utils) 5.2.2**
For [#8859 ](https://github.com/tldr-pages/tldr/issues/8859)

### CODE REVIEW

File: pages/common/xzcat.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# xzcat
+
+> A general purpose data compression tool similar to gzip and bzip2 syntaxes.
+> Compresses or decompresses .xz and .lzma files.
+> More information: <https://www.unix.com/man-page/linux/1/xzcat/>.
```
Comment (navarroaxel) [inline pages/common/xzcat.md:5]:
```suggestion
> More information: <https://manned.org/xzcat>.
```

File: pages/common/xzcat.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+- Force compress:
+
+`xzcat --compress {{path/to/file}}`
+
+- Force decompress:
```
Comment (navarroaxel) [inline pages/common/xzcat.md:11]:
```suggestion
- Decompress a file:
```

File: pages/common/xzcat.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`xzcat --decopress {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/xzcat.md:13]:
```suggestion
`xzcat --decopress {{path/to/file.xz}}`
```

File: pages/common/xzcat.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# xzcat
+
+> A general purpose data compression tool similar to gzip and bzip2 syntaxes.
+> Compresses or decompresses .xz and .lzma files.
+> More information: <https://www.unix.com/man-page/linux/1/xzcat/>.
+
+- Force compress:
```
Comment (navarroaxel) [inline pages/common/xzcat.md:7]: Perhaps a better explanation
```suggestion
- Compress a file then delete it:
```

File: pages/common/xzcat.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+`xzcat --compress {{path/to/file}}`
+
+- Force decompress:
+
+`xzcat --decopress {{path/to/file}}`
+
+- Test compressed file integrity:
```
Comment (navarroaxel) [inline pages/common/xzcat.md:15]:
```suggestion
- Test the integrity of a specific archive:
```

File: pages/common/xzcat.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+`xzcat --test {{path/to/file}}`
```
Comment (navarroaxel) [inline pages/common/xzcat.md:17]:
```suggestion
`xzcat --test {{path/to/archive.xz}}`
```

File: pages/common/xzcat.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+- List information about .xz file:
```
Comment (navarroaxel) [inline pages/common/xzcat.md:19]:
```suggestion
- List the contents of a specific archive:
```

File: pages/common/xzcat.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+
+`xzcat --list {{path/to/file}}`
+
+- Keep/Avoid deleting input files:
```
Comment (navarroaxel) [inline pages/common/xzcat.md:23]:
```suggestion
- Compress a file without delete it:
```

File: pages/common/xzcat.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
+# xzcat
+
+> A general purpose data compression tool similar to gzip and bzip2 syntaxes.
```
Comment (navarroaxel) [inline pages/common/xzcat.md:3]:
```suggestion
> Data compression tool similar to `gzip` and `bzip2` syntax.
```

File: pages/common/xzcat.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+> Compresses or decompresses .xz and .lzma files.
```
Comment (navarroaxel) [inline pages/common/xzcat.md:4]:
```suggestion
> Compresses or decompresses `.xz` and `.lzma` files.
```

**Comment by navarroaxel** [comment]
Remember that you can use this GitHub feature <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request> to apply all the suggestions in 1 commit.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/xzcat.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+- List the contents of a specific archive:
+
+`xzcat --list {{path/to/file}}`
+
+- Compress a file without delete it:
+
+`xzcat --keep {{path/to/file}}`
+
+- Compress a file overwriting the output archive and decompress links:
```
Comment (navarroaxel) [inline pages/common/xzcat.md:27]: @SethFalco could you please help here?
```suggestion
- Compress a file overwriting the output archive compressing the target of symbolic links:
```

**Comment by TerribleCodes** [comment]
> LGTM, Thanks for your contributi

**Comment by Unickorn** [comment]
Hmm, isn't `xzcat` an alias of `xz --decompress --stdout`? (see: https://manned.org/xzcat)

**Comment by navarroaxel** [comment]
Oh, so the default behavior is different.

In this case the `xzcat` should be an alias page. Maybe we can use this PR to use long arguments in the `xz` page, or update the examples

**Comment by TerribleCodes** [comment]
> Oh, so the default behavior is different.
> 
> In this case the `xzcat` should be an alias page. Maybe we can use this PR to use long arguments in the `xz` page, or update the examples

Anything else I can do with this PR?
Like adding more details?

**Comment by navarroaxel** [comment]
@TerribleCodes let's start using long arguments in `xz` then we can improve some example descriptions based on the work we did here.

**Comment by TerribleCodes** [comment]
> @TerribleCodes let's start using long arguments in `xz` then we can improve some example descriptions based on the work we did here.

Should I close this PR since it's already documented? 🤔

**Pull request closed without merging**